system/Session/Session.php

```diff
@@ -232,7 +291,7 @@ protected function configure()
 	 */
 	protected function initVars()
 	{
-		if ( ! empty($_SESSION['__ci_vars']))
+		if (!empty($_SESSION['__ci_vars']))
```

Comment:
Typically, I've been keeping a space after the `!`. I don't know if there's a styleguide reference to that, but it would keep it consistent with the rest of the code.